doc/getting_started/getting_started.rst

```diff
@@ -20,22 +20,73 @@ Follow one of the following guides for your host operating system.
    macOS <installation_mac.rst>
    Windows <installation_win.rst>
 
-Clone the Zephyr Repository
-***************************
+.. _get_the_code:
+
+Get the source code
+*******************
+
+Zephyr's multi-purpose :ref:`west` tool lets you easily get the Zephyr project
+source code, instead of manually cloning the Zephyr repos along with west
+itself.
 
-To clone the Zephyr source code repository from GitHub:
+Bootstrap west
+==============
+
+Install the bootstrapper for Zephyr's command-line tool, :ref:`west` in a
+shell or ``cmd.exe`` prompt:
 
 .. code-block:: console
 
-   git clone https://github.com/zephyrproject-rtos/zephyr
+   # Linux
+   pip3 install --user west
+
+   # macOS and Windows
+   pip3 install west
+
+.. note::
+   See :ref:`gs_python_deps` for additional clarfication on using the
+   `--user` switch.
+
+Additional information about west's structure can be found
+:ref:`in the relevant west documentation section <west-struct>`.
+
+Clone the Zephyr Repositories
+=============================
+
+Clone the Zephyr source code repositories from GitHub using the ``west`` tool:
+
+.. code-block:: console
+
+   west init zephyrproject
+   cd zephyrproject
+   west clone
+
+.. note::
+   You can replace :file:`zephyrproject` with the folder name of your choice.
+   West will create the named folder if it doesn't already exist.
+   If no folder name is specified, west initializes the current
+   working directory.
+
+.. note::
+   If you had previously cloned the zephyr repository manually using Git,
+   move the cloned :file:`zephyr/` repository to an empty enclosing folder
+   (for example :file:`zephyrproject/zephyr/`), and
+   from this empty enclosing folder :file:`zephyrproject/`
+   run ``west init`` and then ``west clone``.
+
+This will create a full Zephyr installation, cloning west itself along with any
```

Comment:
I chose this word because `west init` does more than just fetching Zephyr. It actually installs `west` itself in `<folder>/west/west` alongside the manifest. So in essence you are installing the `west` tool (`pip` only installs the bootstrapper) and then fetching all the repositories required (right now only one, but multiple in the future).

Reply:
> It actually installs west itself  in `<folder>/west/west`

So it clones West and Zephyr into that folder? My point is that here you don't "install" anything in the way people understands "installing". (while pip does install things in the way people understands installing)

Reply:
agree with @aescolar here, this is not a "full Zephyr installation", maybe 'zephyr development environment setup"?